civihr_default_theme/scss/partials/_form.scss

```diff
@@ -405,3 +405,15 @@
         width: 50%;
     }
 }
+
+// This is a temporary copied class from Shoreditch theme
+// @TODO remove this style below as soon as Shoreditch theme is applied
+// Required mark (*)
+.required-mark {
```

Comment:
Question: how come the form has a .required-mark element, but doesn't add the * symbol directly? Can that be explained in the technical details?

Reply:
I updated the technical details, the updated code was manually written and it was missing the class/markup for the star.